### ISSUE TITLE
php-twig2 is a (missing) phpMyAdmin-4.9.4-1.el7.remi.noarch dependency?

### ISSUE THREAD

**Issue by berkenpies** [remicollet/remirepo#135]
We discovered that php-twig2 seems to be a dependency for phpMyAdmin-4.9.4-1.el7.remi.noarch.

The file usr/share/phpMyAdmin/vendor/autoload.php contains a reference to the package:
```
    '/usr/share/php/Psr/Container/autoload.php',
    [
        '/usr/share/php/Twig2/autoload.php',
        '/usr/share/php/Twig/autoload.php',
    ],
```
Without the php-twig2 package, phpMyAdmin (PMA) seems to be working although you're not able to create tables (it will generate a 500 error instead). PMA worked fine after installing the php-twig2 package.

Not sure if it affectes OS's, we're seeing it on RHEL 7.

Please let me know if you need (more) information. :)

**Comment by remicollet** [comment]
This code in the package allow both twig v1 and v2

From upstream composer.json for v4.9
`        "twig/twig": "^1.34 || ^2.0",`

And upstream tarball use version 1.42.4

Version 1 is mandatory to keep compatibility with PHP 5 which is still supported by PMA 4.9

If you have a 500 error, you must also have something recording in the log file.
Can be an upstream issue.

BTW, See #125 and #130 

If you can install php-twig2, this mean you have PHP 7, so I recommend you test PMA 5.0.1 (in remi-test), which requires twig v2.

**Comment by berkenpies** [comment]
> Can be an upstream issue.
> If you can install php-twig2, this mean you have PHP 7

We are actually using PHP 7.x. Decided to report the issue since upstream 4.9. tarball was working fine for us. Therefor I assumed it had to be a problem with this specific package. :)

> If you have a 500 error, you must also have something recording in the log file.

We actually found an error (not recorded, sorry) in the log files which was template related.

> I recommend you test PMA 5.0.1 (in remi-test), which requires twig v2.

As of today we rolled out PMA 5.x on our systems which does have a requirement for php-twig2. This means it's no longer an issue for us.

I guess the issue can be closed now?
